### DISCUSSION TITLE
Make sure to update the Visual Studio 2022 extension to the latest 5.3.x version

### DISCUSSION THREAD

**Discussion by jeromelaban** [unoplatform/uno#17670] [General]
If you're using Visual Studio 2022, make sure to upgrade our latest [VS extension](https://aka.platform.uno/vs-extension-marketplace), either through downloading it then manually then double-clicking on the vsix file, or by going in the **Tools**, **Manage Extensions** menu, then clicking the update button:

<img width="311" alt="image" src="https://github.com/user-attachments/assets/1844599f-be41-4a33-89c4-422ecab011cf">

You may otherwise get errors like [`Could not resolve SDK "Uno.Sdk"`](https://platform.uno/docs/articles/common-issues-all-ides.html#could-not-resolve-sdk-unosdk) when creating Uno Platform projects.

**Comment by agneszitte** [comment]
Related documentation:
- https://platform.uno/docs/articles/migrating-from-previous-releases.html#uno-platform-53
- https://platform.uno/docs/articles/common-issues-all-ides.html#could-not-resolve-sdk-unosdk
